assets/js/modules/search-console/datastore/service.js

```diff
@@ -53,6 +55,19 @@ export const selectors = {
 		}
 		return addQueryArgs( baseURI, queryArgs );
 	} ),
+
+	/**
+	 * Checks whether the property is a Search Console Domain Property.
+	 *
+	 * @since 1.18.0
```

Comment:
Minor but important detail: new `@since` annotations need to always use the literal placeholder `n.e.x.t` since the release version is always unknown. This is already inaccurate though as the most recent release is `1.19.0`, so this wouldn't be released before `1.20.0` but still the placeholder should be used instead as the actual version is applied when the release is prepared.